src/proxy/inbound.rs

```diff
@@ -345,7 +342,7 @@ impl Inbound {
                 };
                 let connection_metrics = traffic::ConnectionOpen {
                     reporter: Reporter::destination,
-                    source,
+                    source: source.map(|wl| wl),
```

Comment:
**question:** what is this intended to do? It's iterating through source, grabbing each wl and not mutating the wl? Was it intended to clone or mutate them somehow?

Reply:
Ah, this was the result of some intermediate changes. Removed the map.